Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'mongoid', '6.1.0'
 gem 'mongoid-history', '0.6.1'
 gem 'mongoid_userstamp', git: 'https://github.com/DCAFEngineering/mongoid_userstamp.git', branch: 'master'
 gem 'mongo_session_store', '>= 3.1.0'
-gem 'mongoid-enum', git: 'https://github.com/DCAFEngineering/mongoid-enum', branch: 'master'
+gem 'enumerize'
```

Comment:
This is awesome, do you know if they are going to support Rails 5.1 anytime soon? The dependency on their end has it pinned to 5.0.0 from what I can tell.